### PR TITLE
Added know more link which redirects to event page

### LINKED ISSUE #196: Add a "Know More" link in the notice corner
### Description
<write your description> I would suggest that here should be a `Know More` link below the events list in the notice corner on the home page that should redirect to the events page.

### Screenshots/testimonials
<place your screenshots or testimonials here if applicable>

Screenshot (46)_LI

### PR DESCRIPTION
**Pull request by spirtskhalava**
Hello,
I implemented this issue and would be glad if you merge it.

### Fixes <https://github.com/houseofgeeks/hg/issues/196>
Added know more link which redirects to event page
<Do not delete this template>
Description: <provide a description> The title should be short and relevant.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>

### CODE REVIEW

File: client/src/components/Home/AboutUs/Event/Event.js [hunk 1]
```diff
@@ -79,6 +79,16 @@ function Event({ page }) {
...
+              <div className="know-more">
+                    <a
+                        rel="noopener noreferrer"
+                        href={"/event"}
+                        className="item-title"
+                      >
+                     Know More..
+                      </a>
+                    
+                  </div>
```
Comment (anktrj) [inline client/src/components/Home/AboutUs/Event/Event.js]: Just a little tweak, make the indentation to 2 spaces!

Comment (anktrj) [reply client/src/components/Home/AboutUs/Event/Event.js]: Also, delete the .DS_Store file

**Review by ankiiitraj — CHANGES_REQUESTED** [review]

**Comment by ankiiitraj** [comment]
small tweaks and you're good to go!

File: client/src/components/Home/AboutUs/Event/Event.js (hunk 1 above)
Comment (ankiiitraj) [reply client/src/components/Home/AboutUs/Event/Event.js:91]: use <Link> instead of <a>

Comment (spirtskhalava) [reply client/src/components/Home/AboutUs/Event/Event.js]: Do you mean indentation the whole block?

**Comment by ankiiitraj** [comment]
I didn't saw this that time!
use `<Link>` instead of `<a>`

**Comment by spirtskhalava** [comment]
> I didn't saw this that time!
> use `<Link>` instead of `<a>`

Done :grin:

**Review by ankiiitraj — APPROVED** [review]

**Comment by ankiiitraj** [comment]
@all-contributors please add @spirtskhalava for code

**Pull request merged**